core/src/main/scala/scalaz/package.scala

```diff
@@ -113,6 +113,8 @@ package object scalaz {
   type <~[+F[_], -G[_]] = NaturalTransformation[G, F]
   type ~~>[-F[_,_], +G[_,_]] = BiNaturalTransformation[F, G]
 
+  type ===[A,B] = Leibniz[⊥, ⊤, A, B]
```

Comment:
there is already a `Leibniz.===`,  I think you should be able to just import it where it is needed without making an alias here.

Reply:
That was my original approach. However, I decided not to continue with that approach because the other type aliases used through scalaz are not imported (I assume) because their type aliases are also defined here. I also read [this part in CONTRIBUTING.md](https://github.com/scalaz/scalaz/blob/series/7.3.x/CONTRIBUTING.md#hierarchy) that made me wonder whether I should refrain from doing so.

So, I was following what I thought was the example in the rest of the code. I can import the `Leibniz.===` if that's desired. If so, is there a specific place where the imports should go / not go?

Reply:
👍 to putting `===` here. At least, I use it more than (say) `BiNaturalTransformation`, which gets a top-level alias. 

I would say that one should point to the other, though. Maybe deprecate `Liskov.===`  and make it an alias of `scalaz.===`?

Reply:
The downside of deprecating the `Leibniz.===` alias is that all type signatures of `A === B` within `Leibniz.scala` now need to be transformed to `===[A, B]` to prevent an error being thrown when I run `sbt compile`.

Reply:
what if you changed `import Leibniz._` there to `import Leibniz.{=== => _, _}`?

Reply:
Oh, duh! On it!

Reply:
> most of them are in the companion. I don't think it's a huge downside at all.

Correct. I should have been clearer.